pom.xml

```diff
@@ -34,7 +34,7 @@
     <groupId>org.opensearch</groupId>
     <artifactId>opensearch-security</artifactId>
     <packaging>jar</packaging>
-    <version>1.0.1.0</version>
```

Comment:
Does [line#106](https://github.com/opensearch-project/security/blob/bbc650c9bbc8b10ed453277fc6a5246c273c8e29/pom.xml#L106) need to be changed?

Reply:
We had such a long debate on it :) I changed it, you're OK with it @vrozov?

Reply:
The `<tag>` can be changed later during release process or it can be changed to `HEAD` now.